docs/assistant/cloud/libraries/adding-assets/assistant-plugin.md

```diff
@@ -0,0 +1,73 @@
+---
+id: assistant-plugin
+title: Adding Assets via the Assistant plugin
+sidebar_label: Assistant Plugin
+---
+
```

Comment:
Needs an intro that explains why the first procedure is "Create a Library"

Reply:
Intro:

In Assistant Pro, all of our assets are organized into libraries.